src/TodoList.js

```diff
@@ -1,58 +1,77 @@
-import React, {useState} from "react";
-import AddTodoForm from "./AddTodoForm";
-import TodoListItem from "./TodoListItem"
-
+import axios from 'axios';
+import React, { useState, useEffect } from 'react';
+import AddTodoForm from './AddTodoForm';
+import TodoListItem from './TodoListItem';
 
 //function to assemble and dissemble list: checkbox, title, fave, edit, trash
 
-const TodoList = ({ useSemiPersistentState, todoTitle }) => {
-
-  const [todoList, setTodoList] = useSemiPersistentState();
+const TodoList = ({ todoTitle }) => {
+  //const [todoList, setTodoList] = useSemiPersistentState();
   const [checked, setChecked] = useState([]);
- 
+  const [todoList, setTodoList] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    axios
+      .get('http://localhost:3001/api/v1/todos?list=636ecac0e0a63a39c7e6217a', {
+        headers: {
+          Authorization:
+            'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiI2MzZlYmUzNGUwYTYzYTM5YzdlNjIxNzMiLCJ1c2VybmFtZSI6Imxpc2FubmEiLCJpYXQiOjE2NjgyMDQ0NTEsImV4cCI6MTY2ODI5MDg1MX0.yvqj2N1E_2DvuPE-5RbW59zjDAWE0SqTSY0xTGNNZ84',
+          'Content-Type': 'application/json',
+          accept: '*/*',
+        },
+      })
```

Comment:
by the way this is alsp secre. we dont want to push it to github. it is better to save in .env then reference it from there.